lms/static/sass/discussion/elements/_actions.scss

```diff
@@ -141,18 +141,15 @@
 
     .action-label {
       @extend %t-copy-sub2;
-      display: inline-block;
+      display: none;
```

Comment:
This may be an a11y issue - previously this was `opacity: 0` and now it's `display: none`, IIRC screenreaders won't read `display: none` elements?

(the real fix would be to use the `.sr-only` class here, as it seems like that's what this was trying to achieve)